packages/libs/types/src/Predicate.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @alpha
+ */
+export const Predicate = {
+  keysAll: 'keys-all',
+  keysTwo: 'keys-2',
+  keysAny: 'keys-any',
+} as const;
+
+/**
+ * @alpha
+ */
+export type TPredicate =
```

Comment:
why string | undefined ?
I think its better to have simple type here
'keys-all' | 'keys-2' | 'keys-any'

Reply:
this whole type means "string" | undefined.

Reply:
true, I think we should drop `string`. It was a suggestion from @alber70g , but maybe I understood it wrong.